it/test/uk/gov/hmrc/tradergoodsprofiles/controllers/GetRecordControllerIntegrationSpec.scala

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tradergoodsprofiles.controllers
+
+import org.mockito.MockitoSugar.{reset, when}
+import org.scalatest.BeforeAndAfterEach
+import org.scalatestplus.mockito.MockitoSugar.mock
+import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import play.api.Application
+import play.api.http.Status.{FORBIDDEN, INTERNAL_SERVER_ERROR, OK, UNAUTHORIZED}
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.Json
+import play.api.libs.ws.WSClient
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import uk.gov.hmrc.auth.core.AffinityGroup.Agent
+import uk.gov.hmrc.auth.core.{AuthConnector, Enrolment, InsufficientEnrolments}
+import uk.gov.hmrc.tradergoodsprofiles.controllers.support.AuthTestSupport
+import uk.gov.hmrc.tradergoodsprofiles.services.DateTimeService
+
+import java.time.Instant
+
+class GetRecordControllerIntegrationSpec
+  extends PlaySpec
+    with GuiceOneServerPerSuite
+    with AuthTestSupport
+    with BeforeAndAfterEach {
+
+  private lazy val wsClient: WSClient = app.injector.instanceOf[WSClient]
+  private lazy val dateTimeService = mock[DateTimeService]
+  private lazy val timestamp = Instant.now
+  private val url = s"http://localhost:$port/goods-profiles/$eroiNumber/records"
```

Comment:
This is missing the "customs/traders/" part of the url. Is this an issue?

Reply:
for local test customer/traders is not needed. this will be put automatically in in dev, qa and other env.